recipes/pato/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e  # exit when any command fails
+
+echo -e "\n### INSTALLING PATO ###\n"
+rm -rf volume
+mkdir volume
+hdiutil attach -mountpoint volume pato_releases_conda.sparsebundle
```

Comment:
Is there a reason we are not building from source?

Reply:
PATO uses a modified version of an older release of OpenFoam, which needs to be patched in multiple ways. I will double check with the main developer of PATO (@jmeurisse) if building from source could be a viable option.

Reply:
We are building from source. The Mac OS installation needs to happen in a disk image.